timeboost-sequencer/src/decrypt.rs

```diff
@@ -360,54 +372,54 @@ impl Decrypter {
     }
 }
 
+fn translate_addr(c: AddressableCommittee) -> AddressableCommittee {
+    let committee = c.committee().clone();
+    let shifted_entries = c
+        .entries()
+        .map(|(pk, dh, addr)| {
+            let dec_port = addr.port().saturating_add(1000);
```

Comment:
The decrypter should be `DECRYPTER_PORT_OFFSET` away from the the public base address.